spec/js/views/app_view_spec.js

```diff
@@ -1,16 +1,21 @@
 define([
   'views/app_view',
+  'collections/todos',
   'ejs'
-], function (Index) {
+], function (Index, ToDoCollection) {
   var instance, subject;
 
   beforeEach(function () {
-    instance = new Index({el: $('body')});
+    instance = new Index({el: $('body'), collection: new ToDoCollection()});
     subject = function () {
       return instance;
     };
   });
 
+  afterEach(function () {
+    subject().$el.empty();
```

Comment:
Better use `instance.$el.empty()`. Subject may be overwritten in other `beforeEach` for child contexts